### DISCUSSION TITLE
Release 3.0.0 timeline?

### DISCUSSION THREAD

**Discussion by jhermsen** [simpleidserver/SimpleIdServer#412] [General]
Hi,

Started playing a bit with the 3.0.0 IdServer.
Really like the more consolidated approach. Also noticed there is quite some work still to do.

So now I'm just wondering if there is any timeline for the 3.0.0 release.

Thanks,
John

**Comment by simpleidserver** [comment]
Hello,

A first pre-release (3.0.0-rc1) package will be published the `15 march`, following features still need to be implemented :

* Manage API Resources : IN PROGRESS
* Update the website to easily create clients
** Machine client (console application, windows service etc...)
** Web application (ASP.NET CORE etc...)
** Native application (mobile application...)
** External Device : Use external device (Back Channel Authentication)
* Edit the client properties
* Add user.
* Add authentication in UI.

Most of the work consists to update the UI :)

In the next release (3.0.1-rc1), following features will be added
* `Manage realm`: Create separate space where you can manage users, scopes, clients etc.. This concept can be used to create one realm per environment (TST, VAL etc...).
** Backend implementation is DONE.
* Update the UI to manage external authentications.
** Backend implementation is DONE.

KR,

SID

**Comment by simpleidserver** [comment]
Hello,

The release `3.0.0` is available.

KR,

SID

**Reply by jhermsen** [reply]
Great, thanks for the update. 
I'm going to try it out asap!
